modules/search/class-jetpack-instant-search.php

```diff
@@ -0,0 +1,367 @@
+<?php
+/**
+ * Jetpack Search: Instant Front-End Search and Filtering
+ *
+ * @since 8.3.0
+ * @package jetpack
+ */
+
+use Automattic\Jetpack\Connection\Client;
+use Automattic\Jetpack\Constants;
+
+class Jetpack_Instant_Search extends Jetpack_Search {
+
+	/**
+	 * Jetpack_Instant_Search constructor.
+	 *
+	 * @since 5.0.0
+	 *
+	 * Doesn't do anything. This class needs to be initialized via the instance() method instead.
+	 */
+	protected function __construct() {
+	}
+
+	/**
+	 * Loads the php for this version of search
+	 *
+	 * @since 8.3.0
+	 */
+	public function load_php() {
+		require_once dirname( __FILE__ ) . '/class.jetpack-search-template-tags.php';
+		require_once JETPACK__PLUGIN_DIR . 'modules/widgets/search.php';
+		require_once dirname( __FILE__ ) . '/class.jetpack-search-template-tags.php';
```

Comment:
Are we intentionally requiring `class.jetpack-search-template-tags.php` twice? If so, why?